src/fluree/db/nameservice/proto.cljc

```diff
@@ -8,13 +8,11 @@
   (-subscribe [nameservice ledger-alias callback] "Creates a subscription to nameservice(s) for ledger events. Will call callback with event data as received.")
   (-unsubscribe [nameservice ledger-alias] "Unsubscribes to nameservice(s) for ledger events")
   (-sync? [nameservice] "Indicates if nameservice updates should be performed synchronously, before commit is finalized. Failure will cause commit to fail")
-
-  (-exists? [nameservice ledger-address] "Returns true if ledger exists (must have had at least one commit), false otherwise")
   (-ledgers [nameservice opts] "Returns a list of ledger aliases registered with this nameservice")
 
   (-close [nameservice] "Closes all resources for this nameservice")
 
   (-alias [nameservice ledger-address] "Given a ledger address, returns ledger's default alias name else nil, if not avail")
-  (-address [nameservice ledger-alias key] "Returns address/iri for provided ledger alias specific to the connection type"))
+  (-address [nameservice ledger-alias key] "Returns full nameservice address/iri which will get published in commit. If 'private', return nil."))
```

Comment:
What is `private`? I'm not familiar with that semantic.

Reply:
private means the nameservice address is not published in the commit file - but the local server can still know how to find it.